traja/dataset/pituitary_gland.py

```diff
@@ -64,14 +64,36 @@ def pituitary_ode(w, t, p):
 
 def compute_pituitary_gland_df_from_parameters(downsample_rate,
```

Comment:
Not clear how this is different than `pituitary_ode`. Can we give it a shorter name?